Xamarin.Forms.Xaml.UnitTests/DataTemplateExtension.xaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+
+using Xamarin.Forms.Core.UnitTests;
+
+namespace Xamarin.Forms.Xaml.UnitTests
+{
+	public partial class DataTemplateExtension : ContentPage
+	{
+		public DataTemplateExtension() => InitializeComponent();
+		public DataTemplateExtension(bool useCompiledXaml)
+		{
+			//this stub will be replaced at compile time
+		}
+
+		[TestFixture]
+		class Tests
+		{
+			[SetUp]
+			public void Setup() => Device.PlatformServices = new MockPlatformServices();
+			[TearDown] public void TearDown() => Device.PlatformServices = null;
+
+			[TestCase(true), TestCase(false)]
```

Comment:
It might be better to indicate multiple test with separate lines for each test case.

Reply:
when you can enumerate all the values of a bool in 2 conditions, I read this as "a single test, for all the possible values of bool" vs multiple tests